docs/data/joy/getting-started/templates/sign-in-side/App.tsx

```diff
@@ -0,0 +1,290 @@
+            backgroundColor: 'rgba(19 19 24 / 0.4)',
+          },
+        })}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            minHeight: '100dvh',
+            width:
+              'clamp(var(--Form-maxWidth), (var(--Collapsed-breakpoint) - 100vw) * 999, 100%)',
+            maxWidth: '100%',
+            px: 2,
+          }}
+        >
+          <Box
+            component="header"
+            sx={{
+              py: 3,
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+            }}
+          >
+            <Typography
+              fontWeight="lg"
+              startDecorator={
+                <Box
+                  component="span"
+                  sx={{
+                    width: 24,
+                    height: 24,
+                    background: (theme) =>
+                      `linear-gradient(45deg, ${theme.vars.palette.primary.solidBg}, ${theme.vars.palette.primary.solidBg} 30%, ${theme.vars.palette.primary.softBg})`,
+                    borderRadius: '50%',
+                    boxShadow: (theme) => theme.shadow.md,
+                    '--joy-shadowChannel': (theme) =>
+                      theme.vars.palette.primary.mainChannel,
+                  }}
+                />
+              }
+            >
+              Logo
+            </Typography>
+            <ColorSchemeToggle />
+          </Box>
+          <Box
+            component="main"
+            sx={{
+              my: 'auto',
+              py: 2,
+              pb: 5,
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 2,
+              width: 400,
+              maxWidth: '100%',
+              mx: 'auto',
+              borderRadius: 'sm',
+              '& form': {
+                display: 'flex',
+                flexDirection: 'column',
+                gap: 2,
+              },
+            }}
+          >
+            <div>
+              <Typography component="h2" fontSize="xl2" fontWeight="lg">
+                Welcome back
+              </Typography>
+              <Typography level="body2" sx={{ my: 1, mb: 3 }}>
+                Let&apos;s get started! Please enter your details.
+              </Typography>
+            </div>
+            <form
+              onSubmit={(event: React.FormEvent<SignInFormElement>) => {
+                event.preventDefault();
+                const formElements = event.currentTarget.elements;
+                const data = {
+                  email: formElements.email.value,
+                  password: formElements.password.value,
+                  persistent: formElements.persistent.checked,
+                };
+                alert(JSON.stringify(data, null, 2));
+              }}
+            >
+              <FormControl required>
+                <FormLabel slotProps={{ asterisk: { sx: { display: 'none' } } }}>
```

Comment:
```suggestion
                <FormLabel>
```

Probably it's not that important and we can simplify it, but up to you :)

Reply:
move to use `formClasses.asterisk` instead.